noetic/distribution.yaml

```
@@ -8106,6 +8106,31 @@ repositories:
       url: https://bitbucket.org/qbrobotics/qbmove-ros.git
       version: production-noetic
     status: developed
+  qb_softhand_industry:
+    doc:
+      type: git
+      url: https://bitbucket.org/qbrobotics/qbshin-ros-release.git
+      version: master
```

This is the correct repository for the documentation; this should be:

```suggestion
      url: https://bitbucket.org/qbrobotics/qbshin-ros.git
      version: production-noetic
```